src/presentation/value-objects/rundown-event.ts

```diff
@@ -55,6 +55,12 @@ export interface PieceInsertedEvent extends PartEvent {
   piece: PieceDto
 }
 
+export interface PieceReplacedEvent extends PartEvent {
+  type: RundownEventType.PIECE_REPLACED
+  replacedPiece: PieceDto
+  newPiece: PieceDto
```

Comment:
Do we need to send the entire Piece that was replaced? Wouldn't it be enough to just send the id?

Reply:
A good point, i'll try that out :-)

Reply:
I tried it out and liked it a lot.